text/0000-sawtooth-governance.md

```diff
@@ -0,0 +1,348 @@
+
+After the final comment period, the subteam can make a decision on the RFC. The
+role of the subteam at that point is *not* to reveal any new technical issues
+or arguments; if these come up during discussion, they should be added as
+comments to the RFC, and it should undergo another final comment period.
+
+Instead, the subteam decision is based on **weighing the already-revealed
+tradeoffs against the project's priorities and values** (which the root team is
+responsible for setting, globally). In the end, these decisions are about how
+to weight tradeoffs. The decision should be communicated in these terms,
+pointing out the tradeoffs that were raised and explaining how they were
+weighted, and **never introducing new arguments**.
+
+### Initial Subteams
+
+The following is a proposed list of initial subteams which is subject to change
+at the discretion of the root team, once it has formed.
+
+**Core** - Responsible for the design, architecture, performance, stability,
+ and security of the core Sawtooth platform
+
+**Application SDKs** - Responsible for the design and consistency of
+user-facing SDKs for application development
+
+**Consensus** - Responsible for the supported consensus algorithms and
+the consensus interface
+
+**Seth** - Responsible for the Sawtooth integration with Ethereum
+
+**Community Outreach** - Responsible for writing and hosting documentation,
+managing the main Sawtooth website, promoting Sawtooth to the public, managing
+demos, and providing the community with training and support
+
+**Release Management** - Responsible for release related issues such as
+dependency management, license compliance, version control, and upgrade and
+backwards compatibility
+
+**Continuous Integration** - Responsible for test and build environments and
+deployment artifacts
+
+## Contributor Permission Levels
+
+The Sawtooth community encourages contributions to the project from all
+interested individuals. The project defines three levels of permissions that
+determine what actions a contributor can perform on a particular repository.
+
+A **reviewer** has "read" permission, meaning they can review pull requests.
+
+A **committer** has "write" permission, meaning they can merge pull requests
+once they have been approved.
+
+Any community member can be promoted to a reviewer or committer by a
+maintainer. Reviewers and committers lose their status when it is removed by a
+maintainer. This is usually due to inactivity, but can also be used for
+moderation.
+
+A **maintainer** has permission to approve changes. All pull-requests must be
+approved by at least 2 maintainers before a committer may merge it.
+Maintainers are expected to:
+
+- Carefully review pull requests and leave thoughtful, constructive feedback
+- Ensure the component is up-to-date with patches and security updates
+- Participate in community discussions related to their component
+
+A committer can be promoted to a maintainer by unanimous vote by the existing
```

Comment:
Does unanimous in this context mean everyone votes for or no one votes against? For the root team, this ambiguity is probably fine, but the list of maintainers might grow quite large, and it might no longer be practical to get an affirmative vote from everyone. So for maintainers in particular, I would suggest the "no one votes against" version, but I'd like to clarify what unanimous means either way.

Reply:
I think the implicit assumption here is that the maintainer list, which is on a per-component/per-repository basis, stays relatively small and is only comprised of the individuals with an expert knowledge and understanding of the code-base. If the maintainer list on a repo grows so large that it is impractical to get an affirmative vote from everyone, then I would interpret that as an indication that the group has grown too large. The intent is to make decisions by consensus and it is the responsibility of those in leadership positions to "drive consensus" by keeping people involved.

Reply:
I'd agree with the suggestion by @delventhalz since this is how an ASF PMC also operates, as long as there are no negative votes and at least one binding positive vote by a PMC member, a proposal passes.

Reply:
I am still leaning towards a "no votes against" version of unanimous. If nothing else, maybe people are sick, or otherwise unavailable (though in that case, we probably need to spell out what a reasonable timeframe for a response on a vote is).

More importantly, however we're using the term, I think it is important to explicitly define unanimous.